src/apiCalls.tsx

```diff
@@ -18,7 +18,7 @@ export const getArticles = (topic = 'home'): Promise<CleanedArticle[]> => {
   return (
     fetch(`https://api.nytimes.com/svc/topstories/v2/${topic}.json?api-key=GKUzDD1VY9ssjZ1AGusX3ci6AeoXCaSr`)
     .then(response => checkResponse(response))
-    .then(data => cleanArticles(data.results.slice(0,10)))
+    .then(data => cleanArticles(data.results.slice(0,3)))
```

Comment:
We'll need to remember to update this for deployment.

Reply:
Sure thing--want me to fix that now? How should I update it?